src/SarifWeb/packages.config

```diff
@@ -30,8 +30,9 @@
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
   <package id="popper.js" version="1.14.0" targetFramework="net461" />
   <package id="Respond" version="1.2.0" targetFramework="net452" />
-  <package id="Sarif.Multitool" version="2.3.4" targetFramework="net461" />
-  <package id="Sarif.Sdk" version="2.3.4" targetFramework="net461" />
+  <package id="Sarif.Multitool" version="2.3.6" targetFramework="net461" />
+  <package id="Sarif.Multitool.Library" version="2.3.6" targetFramework="net461" />
```

Comment:
>Library [](start = 31, length = 7)

In theory, we could at this point remove the reference to Multitool. Instead of running Multitool in an external process, the ValidationService could simply do
```
new ValidationCommand(validationOptions).Run();
```
But we don't want to make that change now, because as the code stands, the validation controller returns the external process's stdout and stderr as part of the web response -- and you don't get those from ValidationCommand.